war/src/main/scss/components/_dialogs.scss

```diff
@@ -3,9 +3,7 @@ $jenkins-dialog-padding: 1.3rem;
 .jenkins-dialog {
   border-radius: 0.6rem;
   border: none;
-  background: transparent;
   box-shadow: var(--dialog-box-shadow);
-  backdrop-filter: var(--dialog-backdrop-filter);
```

Comment:
Could we potentially keep the backdrop filter but remove the `brightness()` (which needs to be removed anyway as it's breaking on HDR displays...):

```css
  background-color: color-mix(in srgb, var(--background) 85%, transparent);
  box-shadow: var(--dialog-box-shadow), 0 0 0 100vmax rgba(23, 28, 37, 0.6);
  backdrop-filter: blur(30px);
  animation: jenkins-dialog-animate-in 0.25s cubic-bezier(0, 0.68, 0.5, 1.5);
  overflow: hidden;
  padding: $jenkins-dialog-padding 0 0 0;
  display: flex;
  flex-direction: column;
  gap: $jenkins-dialog-padding;

  &__title {
    font-size: 1.125rem;
    font-weight: 500;
    padding: 0 $jenkins-dialog-padding;
    color: var(--text-color);
  }
```

Something like the above - the blur is still there just drastically toned down (so we still get a little colour bleed, but nothing overtly distracting like it is now). `::backdrop` has been killed in favour of the increased `box-shadow` as backdrop appears behind the dialog, darkening it, whereas box shadow masks as expected.

Reply:
I would prefer that the dialog has a clear one color only background. It should not be influenced by what is behind the dialog in any way.
btw the same applies to drop down menus and tooltips
![image](https://github.com/user-attachments/assets/214d4799-a615-4c9c-bb54-bcb81d3e3b31)
It just looks not good I think